helper/diffPlaces.js

```diff
@@ -0,0 +1,172 @@
+var _ = require('lodash');
+var placeTypes = require('../helper/placeTypes');
+
+/**
+ * Compare the layer properties if they exist.
+ * Returns false if the objects are the same, and throws
+ * an exception with the message 'different' if not.
+ *
+ * @param {object} item1
+ * @param {object} item2
+ * @returns {boolean}
+ * @throws {Error}
+ */
+function isDiffLayer(item1, item2) {
```

Comment:
the method signature of this function is not very intuitive, it's called `isDiffLayer` which I think is a great name if it returns a boolean value.

looking at the calling code it seems as though the return value is not actually used and the functions main purpose is validation.

a bit nit-picky, but would this function name be better as `validateDiffLayer` (which returns nothing and throws on error) **or** `isDiffLayer` (which does not throw and returns bool so the caller can handle the error case)?